#! /bin/sh /usr/share/dpatch/dpatch-run
# Description: Add missing check so the EFI images aren't installed when VBOX_WITH_EFIFW_PACKING is not defined.
# Author: Felix Geyer <debfx-pkg@fobos.de>

@DPATCH@
diff -Nur virtualbox-ose-3.1.51-dfsg.orig/Makefile.kmk virtualbox-ose-3.1.51-dfsg/Makefile.kmk
--- virtualbox-ose-3.1.51-dfsg.orig/Makefile.kmk	2010-04-27 22:25:22.000000000 +0200
+++ virtualbox-ose-3.1.51-dfsg/Makefile.kmk	2010-05-02 14:02:23.333162297 +0200
@@ -362,7 +362,7 @@
 buildserver-efifw-affinity-hack:
 	$(SHELL) $(PATH_DEVTOOLS)/bin/efi_firmware.sh --cmd fetch --filename $(buildserver-efifw_0_OUTDIR)/VBoxEfiFirmware.zip.tmp
  endif
-else
+else ifdef VBOX_WITH_EFIFW_PACKING
 #
 # Or from the local copy
 #
